mapstory/apps/thumbnails/tasks.py

```diff
@@ -57,13 +58,37 @@ def run_process(self, args, env={}, timeout=66):
 
         return result, out, err  # None if fail
 
+    # add geoserver user/password to a request
+    def request_geoserver_with_credentials(self, url):
+        _user = settings.OGC_SERVER['default']["USER"]
+        _password = settings.OGC_SERVER['default']["PASSWORD"]
+
+        http_client = httplib2.Http()
+        http_client.add_credentials(_user, _password)
+        http_client.add_credentials(_user, _password)
+
+        _netloc = urlparse(url).netloc
+        http_client.authorizations.append(
+            httplib2.BasicAuthentication(
+                (_user, _password),
+                _netloc,
+                url,
+                {},
+                None,
+                None,
+                http_client
+            )
+        )
+        resp, content = http_client.request(url)
+        return content
+
```

Comment:
Can you drop the leading underscores from these? I think that only means something for object instance vars.

Reply:
I copied this from else where - changed.